### DISCUSSION TITLE
CefSharp for crossplatform app

### DISCUSSION THREAD

**Discussion by lofti198** [cefsharp/CefSharp#4743] [Q&A]
Choosing technology stack to develop specialized webbrowser. Wanna use Xamarin  + Cefsharp. Is it available for Xamarin or in which way I could use it for building crossplatform app?

**Comment by amaitland** [comment]
> Is it available for Xamarin

No

`CefSharp` is written in `VC++`. Only runs on windows.

If you need a cross platform browser you'll need to look for a different library.
